Add Decision 015 for Adopting IEEE Style for bib references, closes #671

alxlo opened the pull request.

gernotstarke reviewed (changes requested):
thanx Alex, a small request: Could you please provide 1-2 examples of these references within the ADR please? That provides a bit more comfort to readers, so they don't have to read the extensive IEEE docs.

alxlo commented:
I've added two examples in the decisions and updated the README.md. There should be plenty of further examples in the references section in a short while. ;-)
